docs2/site/docs/guides/known-issues.md

```diff
@@ -144,21 +144,23 @@ Field<StringGraphType>("convertEnumListToString")
 
 ### Can custom scalars serialize non-null data to a null value and vice versa?
 
-No, that is not currently possible, as returning null from `Serialize`, `ParseLiteral` or `ParseValue`
```

Comment:
😮 misinformation)

Reply:
It was correct when I wrote it originally. We have updated the project and neglected to update the FAQ

Reply:
I remember something.